js/main.js

```diff
@@ -25,6 +95,37 @@ $form.addEventListener('submit', function (event) {
   data.nextEntryId++;
 
   data.entries.unshift(newObject);
-  $form.reset();
-  $preview.setAttribute('src', 'images/placeholder-image-square.jpg');
-});
+
+  var $container = document.createElement('div');
+  $container.className = 'container';
+
+  var $row = document.createElement('div');
+  $row.className = 'row';
+  $container.appendChild($row);
+
+  var $columnHalf1 = document.createElement('div');
+  $columnHalf1.className = 'column-half';
+  $row.appendChild($columnHalf1);
+
+  var $imgContainer = document.createElement('div');
+  $imgContainer.className = 'img-container';
+  $columnHalf1.appendChild($imgContainer);
+
+  var $img = document.createElement('img');
+  $img.setAttribute('src', newObject.photo);
+  $imgContainer.appendChild($img);
+
+  var $columnHalf2 = document.createElement('div');
+  $columnHalf2.className = 'column-half';
+  $row.appendChild($columnHalf2);
+
+  var $p1 = document.createElement('p');
+  $p1.textContent = newObject.title;
+  $columnHalf2.appendChild($p1);
+
+  var $p2 = document.createElement('p');
+  $p2.textContent = newObject.notes;
+  $columnHalf2.appendChild($p2);
+
+  $ul.prepend($container);
```

Comment:
You have this in 2 places. There should be a single function that builds the entry HTML structure and `return`s it. You can then use that function in multiple places.  Building it in multiple places will make it hard to maintain, if you make changes you'll have to remember to change it in both places, not ideal.